src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+
+  &-stars {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+  }
```

Comment:
The class name `&-stars` is not according to BEM methodology. In BEM, elements are denoted by two underscores (__), and modifiers are denoted by two hyphens (--). So, the correct class name should be `&__stars`.